packages/bpk-component-modal/src/BpkModalInner.tsx

```diff
@@ -91,14 +104,14 @@ const BpkModalInner = (props: Props) => {
       transitionTimeout={300}
     >
       <section
-        id={props.id}
-        tabIndex="-1"
+        id={rest.id}
```

Comment:
Do we not need to define this above instead of pulling it out of `rest`? Or is this how required props are supposed to work?

Reply:
no, we can pull them out that's ok!